openjdk/ProblemList_openjdk13-openj9.txt

```diff
@@ -181,6 +180,7 @@ java/net/httpclient/websocket/PendingPongBinaryClose.java https://github.com/ecl
 java/net/httpclient/websocket/PendingPongTextClose.java https://github.com/eclipse/openj9/issues/4298 macosx-all
 java/net/httpclient/websocket/PendingTextPingClose.java https://github.com/eclipse/openj9/issues/4298 macosx-all
 java/net/httpclient/websocket/PendingTextPongClose.java https://github.com/eclipse/openj9/issues/4298 macosx-all
+java/net/CookieHandler/B6791927.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1436	generic-all
```

Comment:
Based on the comment in issue #1436 - the expectation is that this test is fixed, and the fix should be picked up in the tagged branch used by test.  Do you still wish to exclude it?

Reply:
Yes, I excluded it so that it will be easier to see any failing tests coming up to the release (making the pipeline green). I believe the codebase for jdk13 is being held onto until the release so we won't get any new tags until then. I'll just need to remember to unexclude it after the release